htdocs/Language/default.xml

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0"?>
 <pages lang="default" descr="Default" version="0.8.4">
     <page id="general" descr="General Terms">
+        <term>
+            <key>ONE_TEST_PER_PAGE</key>
+            <value>1 Test Per Page</value>
+        </term>
```

Comment:
Rereading my email I see I forgot a step - can you also copy the XML files to the `langdata` folder?

Reply:
Should be resolved now